mlflow/tracking/fluent.py

```diff
@@ -187,6 +187,15 @@ def set_tag(key, value):
     MlflowClient().set_tag(run_id, key, value)
 
 
+def delete_tag(key):
```

Comment:
Attempting to invoke
```
import mlflow
mlflow.delete_tag(...)
```
yields
```
AttributeError: 'module' object has no attribute 'delete_tag'
```

Looks like we aren't exporting the `delete_tag` function to the top-level MLflow module. You can do that by adding an entry here: https://github.com/mlflow/mlflow/blob/9390f44e310faef7f030a75c6421754a4deff951/mlflow/__init__.py#L48

This also seems to imply that we don't have a test that calls `mlflow.delete_tag`; can we add one? `mlflow/tests/tracking/test_fluent.py` seems like a good location.

Reply:
Thanks for flagging this - sorry about missing it. Next commit has a fix now.